apps/chat/README.md

```diff
@@ -9,7 +9,20 @@ The easiest way to get started with messaging in the Amazon Chime SDK is to depl
 - An [AWS account](https://aws.amazon.com/free/) that provides access to AWS services.
 - [Node.js](https://nodejs.org/en/download/) version 14 or higher.
 
-## Deploying the solution
+## Deploying the demo app
+
+You can run deploy.sh script in this folder to automatically deploy the demo app and it's backend.
```

Comment:
- `it's backend` => `its backend`
- Or just end with `deploy the demo app.` since the "app" indicates both frontend and backend.

Reply:
good catch, updated Kyu